packages/monitors/README.md

```diff
@@ -0,0 +1,31 @@
+# @uma/monitors
+
+This package contains four UMA monitor bot implementations: `BalanceMonitor`, `ContractMonitor`, `CRMonitor`, and `SyntheticPegMonitor`. The monitor bots are used to monitor the UMA ecosystem for key events. They have a number of monitor modules built into them that enable real time reporting on key events for a given EMP contract.
+
+For more information about running a monitor bot, see the [docs](https://docs.umaproject.org/developers/bots).
+
+## Installing the package
+
+```bash
+yarn global add @uma/monitors
+```
+
+## Running the monitors
+
+The config below will start up a monitor bot that will: (1) send messages when new liquidations, alerts, or disputes occur and (2) fire if there is large volatility in the synthetic or price of the underlying. It wont report on any wallet or CR monitoring as no params have been defined.
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=your.node.url MNEMONIC="your mnemonic here" monitors --network mainnet_mnemonic
```

Comment:
Have you tested that this is minimal viable config neccessary to run the bots? Also, shouldn't the execution command be `yarn monitors ...`

Reply:
Running some tests this afternoon! Was using the examples from the main readme branch as a starting point.

Reply:
Agreed, do the monitors work with this few params?

@nicholaspai `monitors` alone works when you install the package globally as was done in the previous section.

@pemulis see the thread on my PR here: https://github.com/UMAprotocol/protocol/pull/2356#discussion_r550169440. @chrismaree suggested that we not install these globally, so I removed the `global` from `yarn global add liquidator` and I added `yarn` to the run command, making it `... yarn liquidator --network ...`.

Reply:
I also wasn't sure how to run the serverless orchestration or reporters, so those could use some fleshing out.

Reply:
Working from this section of the docs on the monitor bots, by the way:

https://docs.umaproject.org/developers/bot-param#minimum-viable-monitor-config

Hope to have a better handle on the bots from the review later this week.